code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -21,3 +22,60 @@
 			H.take_overall_damage(1,1)
 		else if (light_amount < 0.2) //heal in the dark
 			H.heal_overall_damage(1,1)
+
+
+/datum/species/shadow/nightmare
+	name = "Nightmare"
+	id = "nightmare"
+	burnmod = 1.5
+	blacklisted = TRUE
+	species_traits = list(NOBREATH,RESISTCOLD,RESISTPRESSURE,NOGUNS,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,PIERCEIMMUNE,NODISMEMBER,NO_UNDERWEAR)
+	mutanteyes = /obj/item/organ/eyes/night_vision/nightmare

```

Comment:
I think @FuryMcFlurry made some new shadowling race sprites, so if you'd like to use those they'd look very much better than the old.

Reply:
i think we've been here